docs/source/zero-code-change-benchmarks.rst

```diff
@@ -1,2 +1,56 @@
-Benchmarks
+cuml.accel: Zero Code Change Acceleration Benchmarks
+====================================================
+
+Training
+--------
+
+One of the biggest advantages of cuML is how much faster training becomes, which is true as well when using the cuML Accelerator, with no code changes!
+
+Algorithms like HDBSCAN, t-SNE, and UMAP often see speedups anywhere from 60× to over 300×, transforming tasks that could take hours or even days into minutes. Even more conventional methods such as KMeans and Random Forests can achieve training speedups of 15× to 80×, while models like LogisticRegression, Lasso, PCA, and Ridge still benefit significantly (typically in the 2×–10× range). The following benchmarks are a summary of the type of speedups that can be seen in datasets that range from 8 to 512 columns.
+
+.. image:: img/overall_speedup.png
+   :alt: Overall speedup
+
+
+Comparison of cuML and cuml.accel
+---------------------------------
+
+One question about cuml.accel is: how does it compare to using cuML directly? The answer varies, but in cases where the models are compared with the same parameters and data sizes, the overhead is quite small when it comes to training models:
+
+.. image:: img/overall_overhead.png
+   :alt: Overall overhead
+
+Two important aspects arise immediately: training typically is quite computationally expensive, so the cost of transferring data from CPU and GPU and the machinery of cuml.accel have overheads that don't affect the acceleration and running time significantly. But even here, one can immediately notice that the overhead is more significant for tasks that are simpler, for example training KNeighbors modeks. There, using cuML directly can be significantly faster if one wants to get the maximum performance of a GPU, though it's important to note that the difference in execution time is the difference of computing in seconds vs miliseconds.
```

Comment:
I'd recommend rewriting this paragraph for simplicity and clarity. It's not obvious to me until partway through the paragraph what it's trying to tell me. Maybe we put the bottom line up front: "Simpler estimators tend to have relatively more overhead with cuml.accel since they already run quite fast." And then we can go on to provide detail about that.